website/src/components/Messages/RenderedMarkdown.tsx

```diff
@@ -83,17 +86,64 @@ const disallowedElements = ["img"];
 const RenderedMarkdown = ({ markdown }: RenderedMarkdownProps) => {
   const { isOpen, onOpen, onClose } = useDisclosure();
   const [link, setLink] = useState<string | undefined>();
+  const [isCopied, setIsCopied] = useState(false);
```

Comment:
This will cause a markdown containing multiple `code` blocks will show a copied state in the same time

Reply:
Will do in the morning, it's pretty late now 😅